config.js

```diff
@@ -53,10 +53,6 @@ module.exports = {
       'restify': path.join(__dirname, 'src/plugins/plugin-restify.js')
     },
 
-    // Valid entries are:
-    // 'express', 'hapi', 'http', 'restify'
-    excludedHooks: [],
```

Comment:
Plugins are now specified through the config. Users can specify in their config precisely which plugins they want.